well-known-bots.json

```diff
@@ -5526,6 +6069,7 @@
     ],
     "pattern": "Google-Site-Verification",
     "addition_date": "2019/12/11",
+    "verification": [],
```

Comment:
There's no verification on this google bot? There might be some others here so please double check the category